### PR TITLE
Add more debug options

### PR DESCRIPTION
**Pull request by m-iwanicki**
Remove code duplication in kas files & add debug kernel config to add  `/proc/config.gz`

### CODE REVIEW

File: kas/debug.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+header:
+  version: 11
+
+local_conf_header:
+  override: |
+    OVERRIDES:append = ":dbg"
```
Comment (DaniilKl) [inline kas/debug.yml:7]: @m-iwanicki, I see some good ideas here. What if we can use this further, for example:

```diff
diff --git a/kas-debug-webkit.yml b/kas-debug-webkit.yml
index 0906254a2e5e..38943a852f2e 100644
--- a/kas-debug-webkit.yml
+++ b/kas-debug-webkit.yml
@@ -4,6 +4,3 @@ header:
   includes:
     - kas-prod-webkit.yml
     - kas/debug.yml
-
-target:
-  - zarhus-base-image-debug
diff --git a/kas-debug.yml b/kas-debug.yml
index 93eeaa006ed1..121304d0c98c 100644
--- a/kas-debug.yml
+++ b/kas-debug.yml
@@ -4,6 +4,3 @@ header:
   includes:
     - kas-prod.yml
     - kas/debug.yml
-
-target:
-  - zarhus-base-image-debug
diff --git a/recipes-zarhus/images/zarhus-base-image-debug.bb b/recipes-zarhus/images/zarhus-base-image-debug.bb
deleted file mode 100644
index d14027002386..000000000000
--- a/recipes-zarhus/images/zarhus-base-image-debug.bb
+++ /dev/null
@@ -1,7 +0,0 @@
-require zarhus-base-image.inc
-
-IMAGE_FEATURES += "debug-tweaks"
-
-IMAGE_INSTALL:append = " \
-    packagegroup-zarhus-dbg \
-"
diff --git a/recipes-zarhus/images/zarhus-base-image.inc b/recipes-zarhus/images/zarhus-base-image.inc
index 3cbb0ab6db97..475026872bc8 100644
--- a/recipes-zarhus/images/zarhus-base-image.inc
+++ b/recipes-zarhus/images/zarhus-base-image.inc
@@ -1,6 +1,7 @@
 SUMMARY = "zarhus image"
 
 IMAGE_FEATURES += "ssh-server-openssh"
+IMAGE_FEATURES:dbg += "debug-tweaks"
 
 LICENSE = "MIT"
 
@@ -9,3 +10,7 @@ inherit core-image
 IMAGE_INSTALL:append = " \
     packagegroup-zarhus-system \
 "
+
+IMAGE_INSTALL:dbg:append = " \
+    packagegroup-zarhus-dbg \
+"
```

By adding this override, we can get rid of the separate recipes for debug images. What do you think?

Comment (m-iwanicki) [reply kas/debug.yml:7]: Have you tested it? If it works then I'm for it as there will be less risk that debug/prod images will get desynced.

Comment (DaniilKl) [reply kas/debug.yml:7]: It does compile but does not work:
```bash
[    0.771854] mmc_host mmc2: Bus speed (slot 0) = 150000000Hz (slot req 150000000Hz, actual 150000000HZ div = 0)
sh: cannot set terminal process group (-1): Inappropriate ioctl for device
sh: no job control in this shell
sh-5.2# [    1.075402] dwmmc_rockchip fe2c0000.mmc: Successfully tuned phase to 207
```

Comment (DaniilKl) [reply kas/debug.yml:7]: The idea is OK, but maybe for another time.

Comment (m-iwanicki) [reply kas/debug.yml:7]: Weird, there were no changes that would stop image from working. Either it would build everything (debug) or only prod part.

Comment (DaniilKl) [reply kas/debug.yml:7]: Yep, wierd.

**Pull request merged**